lib/handbook/addon/docs/template.hbs

```diff
@@ -7,6 +7,7 @@
         {{nav.section 'Development'}}
         {{nav.item 'Quickstart' 'docs.quickstart'}}
         {{nav.item 'Troubleshooting' 'docs.troubleshooting'}}
+        {{nav.item 'Additional Resources' 'docs.resources'}}
```

Comment:
s/R/r

Reply:
Should the title on the page also use this casing?

Reply:
No, page headers are title-case, at least going by osf-style: https://centerforopenscience.github.io/osf-style/language.html#capitalization